internal/table.go

```diff
@@ -3,7 +3,6 @@ package internal
 import (
 	"fmt"
```

Comment:
This file still imports "fmt" package. Make sure only the verbose package is used, otherwise it means that we have missed some lines where we are logging stuff.